### DISCUSSION TITLE
Fastlane : [altool] Error: Unable to upload archive. Failed to get authorization for username and password

### DISCUSSION THREAD

**Discussion by thejeff77** [fastlane/fastlane#21865] [General]
I've setup the .env file as mentioned by @jakansha2001 https://github.com/fastlane/fastlane/issues/21060#issuecomment-1772362069. I've also run fastlane precheck and it says it can login fine.

Despite the above steps being completed, I still get the following:

 [Application Loader Error Output]: Error uploading '/var/folders/rm/gb3cy4bj14n_dr8c6m8c4n2m0000gn/T/ac7a32f8-3bee-43c3-a377-6d6f2cc68144.ipa'.
[Application Loader Error Output]: Unable to upload archive. Failed to get authorization for username 'someschmuck@gmail.com' and password. (
[Application Loader Error Output]: The call to the altool completed with a non-zero exit status: 1. This indicates a failure.

Plus as previously mentioned on that issue thread previously mentioned, all of the [apple systems](https://developer.apple.com/system-status/) are showing as green and up.

Any help with this would be very appreciated! The fastlane process should run this on precheck and also give a helpful link for setup if its setup related, which it doesn't appear to be so far.

Why would precheck say its all fine and dandy but then this would fail like it did?

Should I log another issue? This one seems to be closed.

**Comment by darrin5698** [comment]
Despite setting up the .env file as mentioned and successfully running fastlane precheck, I encounter the following error during the upload process:

vbnet
Copy code
[Application Loader Error Output]: Error uploading '/var/folders/rm/gb3cy4bj14n_dr8c6m8c4n2m0000gn/T/ac7a32f8-3bee-43c3-a377-6d6f2cc68144.ipa'.
[Application Loader Error Output]: Unable to upload archive. Failed to get authorization for username 'someschmuck@gmail.com' and password.
[Application Loader Error Output]: The call to the altool completed with a non-zero exit status: 1. This indicates a failure.
This error persists even though Apple systems are operational, as indicated by their status page. It's perplexing that fastlane precheck reports everything is fine, yet the upload fails with an authorization error.

Could you provide any insights or solutions for this issue? Should I log a new issue since the existing thread seems to be closed? Any assistance would be greatly appreciated. [StarbucksSecretMenu](https://www.starbucks-secretmenu.com)
